posts/2022-12-06-22.0.0.13-beta.adoc

```diff
@@ -0,0 +1,340 @@
+    
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/23355
+// Contact/Reviewer: ReeceNana,arkarkala
+// // // // // // // // 
+[#authorization]
+== Application Authorization 2.1 (Jakarta Authorization 2.1)
+Jakarta Authorization defines a low-level SPI for authorization modules, which are repositories of permissions facilitating subject based security by determining whether a given subject has a given permission, and algorithms to transform security constraints for specific containers (such as Jakarta Servlet or Jakarta Enterprise Beans) into these permissions.
+
+Jakarta Authorization 2.1 specification provides enhancements to the Jakarta Authorization 2.0 specification by the addition of new methods to read permissions.  One such addition is the ability to get the `PolicyConfiguration`, which is used for authorization.  Prior to Jakarta Authorization 2.1, the spec did not have the flexibility to read permissions from the `PolicyConfiguration` and required complex wrapper code to get the `PolicyConfiguration`.
+   
+You can enable the Jakarta Authorization 2.1 feature by using the `appAuthorization-2.1` feature in the server.xml file:
+   
+[source,xml]
+----
+    <featureManager>
+        <feature>appAuthorization-2.1</feature>
+    </featureManager>
+----
+   
+For more information see the https://github.com/jakartaee/specifications/blob/master/authorization/2.1/_index.md[Jakarta Authorization 2.1] specification.    
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/23315
+// Contact/Reviewer: teddyjtorres,ReeceNana
+// // // // // // // // 
+[#security]
+== Application Security 5.0 (Jakarta Security 3.0)
+
+Application Security 5.0 enables support for securing the server runtime environment and applications using Jakarta Security 3.0. The Jakarta Security 3.0 specification allows using built-in authentication mechanisms and identity stores as well as providing your own custom implementations for authenticating users to protected resources. The specification introduces a new built-in authentication mechanism based on the Authorization Code Flow of the OpenID Connect Core 1.0 specification.
+
+Jakarta Security 3.0 allows using the new `@OpenIdAuthenticationMechanismDefinition` annotation to configure a built-in OpenID Connect authentication mechanism to function as an OpenID Connect client, or Relying Party (RP), so that you can use an OpenID Connect Provider (OP) as a single-sign-on identity provider. It also introduces the `OpenIdContext` bean and various APIs for handling the access token, identity token, and refresh token.

```

Comment:
```suggestion
Jakarta Security 3.0 allows using the new `@OpenIdAuthenticationMechanismDefinition` annotation to configure a built-in OpenID Connect authentication mechanism to function as an OpenID Connect client, or Relying Party (RP), so that you can use an OpenID Connect Provider (OP) as a single sign-on identity provider. It also introduces the `OpenIdContext` bean and various APIs for handling the access token, identity token, and refresh token.

```